src/scss/components/modal/_index.scss

```diff
@@ -31,25 +33,26 @@ hx-modal {
   }
 
   > hx-div {
-    --hxPaddingLeft: @space[md];
-    --hxPaddingRight: @space[md];
+    --hxPaddingLeft: #{map-get($space, md)};
+    --hxPaddingRight: #{map-get($space, md)};
+
     flex-grow: 1;
     order: 2;
 
-    #HXDiv.shadyRoot({
-      padding: 0 @space[md];
-    });
+    @mixin hxDiv-shadyRoot{
```

Comment:
`hxDiv-shadyRoot` not properly applied

Please correct all similar updates in this file.

* Use `@include`, not `@mixin`.
* Space before opening `{`

```scss
@include hxDiv-shadyRoot {
   ...
}
```

Reply:
Changes has been done